src/sentry_value.c

```diff
@@ -1326,3 +1326,9 @@ sentry_transaction_remove_data(sentry_value_t transaction, const char *key)
 {
     sentry_span_remove_data(transaction, key);
 }
+
+void
+sentry_transaction_set_name(sentry_value_t tx, const char *name)
+{
+    sentry_value_set_by_key(tx, "transaction", sentry_value_new_string(name));
```

Comment:
this can return an error if tx was not an object...  is it really a good idea to make this void?

Reply:
This problem should solve itself once we have strong types.

Reply:
true, fain enough

Reply:
*fair